<launch>
  <param name="/use_sim_time" value="true"/>
   <param name="/radius" value="0.1"/>
   <param name="/grid_size_x" value="59"/>
   <param name="/grid_size_y" value="59"/>
   <param name="/grid_step" value="0.5"/>
   
   , , 
   <node pkg="ros_demo" type="state_publisher.py" name="state_publisher" output="screen"/>

   <node pkg="scan_point_cloud" type="scan2pcl1" name="scan2pcl1"/>

</launch>
